src/db/migrations/20210307131630-create-article.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Articles', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userId: {
+        type: Sequelize.INTEGER
+      },
```

Comment:
Request: can you add another field `authorName`? Our updated design may require the name to be separate than the user ID, because only the Excos can publish the articles for now, and they would have to name the article themselves.